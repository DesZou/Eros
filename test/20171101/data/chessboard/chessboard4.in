6
o*ooo*
*ooooo
oooo*o
oooooo
o*oooo
ooooo*
