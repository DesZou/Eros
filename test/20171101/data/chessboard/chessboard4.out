1
583126079
486437824
4177920
0
0
0
